plugin.video.vrt.nu/resources/lib/vrtplayer/urltostreamservice.py

```diff
@@ -152,8 +152,10 @@ def _select_stream(self, stream_dict, vudrm_token):
             return self._try_get_drm_stream(stream_dict, vudrm_token)
         elif vudrm_token:
             return streamurls.StreamURLS(*self._select_hls_substreams(stream_dict['hls_aes']))
-        else:
+        elif self._kodi_wrapper.has_inputstream_adaptive_installed():
```

Comment:
So because you cannot install version 1.5.+ without having inputstream adaptive installed this is unessecary

Reply:
I know, this is just to preserve support for the non-drm hls streams. You never know if the drm protected streams will fail some point in the future